comm/src/main/scala/coop/rchain/comm/transport/GrpcTransport.scala

```diff
@@ -56,7 +56,6 @@ object GrpcTransport {
       .flatMap(
         tlr =>
           tlr.payload match {
-            case p if p.isProtocol   => Right(Some(tlr.getProtocol))
             case p if p.isNoResponse => Right(None)
```

Comment:
Please refactor the functions `processResponse` and `transport` to return `CommErr[Unit]`. Why does it still return `CommErr[Option[Protocol]]`? I thought that `Protocol` is gone?

Reply:
I will be removing `CommunicationResponse` and refactoring `PacketHandler` in next PR (small steps)